nautobot/extras/tables.py

```diff
@@ -295,6 +295,7 @@ class DynamicGroupMembershipTable(DynamicGroupTable):
     """Hybrid table for displaying info for both group and membership."""
 
     description = tables.Column(accessor="group.description")
+    actions = ButtonsColumn(DynamicGroup, pk_field="slug", buttons=("edit",))
 
     class Meta(BaseTable.Meta):
         model = DynamicGroupMembership
```

Comment:
UI nitpick - I keep finding it unintuitive that `operator` comes after `name`, as it reads to me as:

```
Group 1      AND
Group 2      OR
Group 3      NOT
  Subgroup A AND
```

i.e. `Group 1 AND Group 2 OR Group 3` when it's actually interpreted as `(all) AND Group 1 OR Group 2 NOT Group 3`, so I think a more intuitive and accurate visual representation would be:

```
AND     Group 1
OR      Group 2
NOT     Group 3
  AND       Subgroup A
```

Reply:
I know you're commenting on the table here, but is that what you mean? Are you thinking this?

<img width="1978" alt="image" src="https://user-images.githubusercontent.com/138052/181666377-a7a0023e-414c-46f1-a17d-59cde2dab534.png">

Reply:
Operator, Name, Weight, Filter, Group Members, Description

🤔

Reply:
<img width="1978" alt="image" src="https://user-images.githubusercontent.com/138052/181787218-7183074a-ba45-4a10-aede-5fbe99d48b6e.png">

Reply:
And the edit view w/t operator first: 
<img width="976" alt="image" src="https://user-images.githubusercontent.com/138052/181787662-e2002b12-9b03-475b-b2c1-6e479749900c.png">

Reply:
I am curious how the `Filter` column displays when the filter is based on say, a dozen distinct attributes rather than a single value of a single attribute, and how that impacts the table rendering as a whole, but that can probably wait for a future PR.